### PR TITLE
Fixed Image Path

### PR DESCRIPTION
**Pull request by PHLAK**
Fixed a relative image path on the Behaviour-Driven Development page

### CODE REVIEW

File: content/docs/bdd/_index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The idea is to make each change small and iterate rapidly, moving back up a leve
 We call these practices _Discovery_, _Formulation_, and _Automation_.
 
 <figure>
-  <img alt="diagram of how the practices fit together" src="../../img/BDD practices diagram (loops).png" width="600px">
+  <img alt="diagram of how the practices fit together" src="../../../static/img/BDD practices diagram (loops).png" width="600px">
```
Comment (mlvandijk) [inline content/docs/bdd/_index.md]:
```suggestion
  <img alt="diagram of how the practices fit together" src="../../../img/BDD practices diagram (loops).png" width="600px">
```

**Review by mlvandijk — CHANGES_REQUESTED** [review]
Thanks for pointing out the pic is broken. 
From the preview ("Show all checks" > "Details") this doesn't seem to work either. Coudl you try the suggestion in review? That works for me when building locally.

File: content/docs/bdd/_index.md (hunk 1 above)
Comment (PHLAK) [reply content/docs/bdd/_index.md]: `static/img/BDD practices diagram (loops).png` is the correct path (see: https://github.com/cucumber/docs.cucumber.io/blob/master/static/img/BDD%20practices%20diagram%20(loops).png).

You can confirm this by going to the `Files changed` tab of this PR and clicking the `Display the rich diff` button where you'll see the expected image.

Comment (PHLAK) [reply content/docs/bdd/_index.md]: Never mind, I understand the issue now. The way the files are hosted by Netlify gives it a different path. Will fix it.

**Review by mlvandijk — APPROVED** [review]
Image is shown correctly in preview now. Thanks for the fix!

**Comment by aslakhellesoy** [comment]
Hi @PHLAK,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by mattwynne** [comment]
Thanks @PHLAK!

**Pull request merged**